src/components/home/NetworkCard.tsx

```diff
@@ -43,6 +43,13 @@ const NetworkCard: React.FC<{ network: Network }> = ({ network }) => {
             suffix={<LinkOutlined />}
           />
         </Col>
+        <Col span={12}>
+          <Statistic
+            title={l('taroNodes')}
+            value={network.nodes.taro.length}
+            suffix={<ApiOutlined />}
```

Comment:
antd doesn't provide a lot of icons to choose from but I think the `DollarOutlined` one makes more sense for Taro since the primary goal of the project is to bring stablecoins to Bitcoin.

Reply:
Done.